<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emprunts_dialog</class>
 <widget class="QDialog" name="emprunts_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background-color : white ;
}

QPushButton
{
background-color : #181DA4 ;
border-radius : 20px ;
color : white ;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>229</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>ID d'emprunt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>504</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Adhérent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_id_adr"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Code d'adhérent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_code_adhr">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Nom d'adhérent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nom_adr">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Nombre d'emprunts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nbrEmp_adr_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>501</width>
     <height>175</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Livre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_id_livre"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Titre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_titre">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_code">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nombre de pages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nbrp">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Auteur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_auteur">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Etat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_etat">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Date d'emprunt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="lineEdit_date_emp"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Date de retour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="lineEdit_date_ret"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>481</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>620</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_init">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/46.png</normaloff>:/new/prefix1/46.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/59.png</normaloff>:/new/prefix1/59.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/45.png</normaloff>:/new/prefix1/45.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/33.png</normaloff>:/new/prefix1/33.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>590</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/42.png</normaloff>:/new/prefix1/42.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/new/prefix1/67.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Emprunts</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
